src/main/java/org/openlmis/integration/dhis2/service/schedule/DynamicCronScheduler.java

```diff
@@ -73,7 +73,7 @@ public void createNewCron(Schedule schedule) {
     CronTrigger cronTrigger = new CronTrigger(periodGeneratorService, periodEnum, offset);
 
     ScheduledFuture<?> newProcess = taskScheduler.schedule(
-        () -> processedDataExchangeService.sendData(schedule), cronTrigger);
+        () -> processedDataExchangeService.sendData(schedule, null, null), cronTrigger);
```

Comment:
sending null, null as parameters is always weird. What's the reason to add these params in method when you send it as null

Reply:
I overloaded the sendData method instead of passing nulls in every call.